Gruntfile.js

```diff
@@ -68,29 +76,37 @@ module.exports = function(grunt) {
 
     browserify: {
       options: {
-        banner: '<%= banner %>'
+        banner: '<%= banner %>',
+        transform: [
+          'babelify'
+        ]
       },
-      dev: {
+      src: {
         src: ['lib/videojs-playlist-ui.js'],
         dest: 'dist/videojs-playlist-ui.js'
```

Comment:
I haven't had issues making use of things as-is in example.html but I'll keep an eye on this.